Pocket/recommendation-api
# feat(Home): Locale is included in Snowplow event [TDP-289]

# Goal
Emit locale as part of Home Snowplow event to allow adding locale as a dimension to reports.

## QA
- Automated tests use Snowplow Micro to confirm that the expected events are emitted and valid according to the schema.

## Deployment
- [x] Deploy https://github.com/Pocket/spec/pull/181
- [ ] Change Dbt models that are dependent on `corpus_slate_lineup_1`

## Reference

Tickets:
* https://getpocket.atlassian.net/browse/TDP-289
